full/coupler_main.F90

```diff
@@ -446,7 +442,8 @@ program coupler_main
   if (check_stocks >= 0) call coupler_flux_init_finish_stocks(Time, Atm, Land, Ice, Ocean_state, &
                                                               coupler_clocks, init_stocks=.True.)
 
-  do nc = 1, num_cpld_calls
+  !> ocean/slow-ice integration loop
+  slow_integration_loop : do nc = 1, num_cpld_calls
```

Comment:
This should be named something like `coupled_timestep_loop` because it’s the coupled timestep